src/input/internal.tsx

```diff
@@ -35,6 +35,8 @@ export interface InternalInputProps
 
   __onDelayedInput?: NonCancelableEventHandler<BaseChangeDetail>;
   __onBlurWithDetail?: NonCancelableEventHandler<{ relatedTarget: Node | null }>;
+
+  __useFormField?: boolean;
```

Comment:
React naming suggests to avoid using `use-` for everything, which is not a hook. This one could be `inheritFormFieldLabels`

Reply:
Love that!